cypress/e2e/awx/resources/workflowVisualizerCRUD.cy.ts

```diff
@@ -0,0 +1,459 @@
+  let workflowJtNode: WorkflowNode;
+
+  before(function () {
+    organization = this.globalOrganization as Organization;
+    project = this.globalProject as Project;
+    cy.awxLogin();
+
+    cy.createAwxInventory({ organization: organization.id })
+      .then((i) => {
+        inventory = i;
+      })
+      .then(() => {
+        cy.createAwxInventorySource(inventory, project).then((invSrc) => {
+          inventorySource = invSrc;
+        });
+        cy.createAwxJobTemplate({
+          organization: organization.id,
+          project: project.id,
+          inventory: inventory.id,
+        }).then((jt) => (jobTemplate = jt));
+      });
+  });
+
+  beforeEach(function () {
+    cy.createAwxWorkflowJobTemplate({
+      organization: organization.id,
+      inventory: inventory.id,
+    }).then((wfjt) => (workflowJobTemplate = wfjt));
+  });
+
+  afterEach(() => {
+    cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate, { failOnStatusCode: false });
+  });
+
+  after(function () {
+    cy.deleteAwxInventory(inventory, { failOnStatusCode: false });
+    cy.deleteAwxInventorySource(inventorySource, { failOnStatusCode: false });
+    cy.deleteAwxJobTemplate(jobTemplate, { failOnStatusCode: false });
+  });
+
+  describe('Workflow Visualizer: Create', () => {
+    it('should render a workflow visualizer view with multiple nodes present', () => {
+      cy.renderWorkflowVisualizerNodesFromFixtureFile(
+        `${workflowJobTemplate.name}`,
+        'wf_vis_testing_A.json'
+      );
+      cy.get('[class*="66-node-label"]')
+        .should('exist')
+        .should('contain', 'Cleanup Activity Stream');
+      cy.get('[class*="43-node-label"]').should('exist').should('contain', 'bar');
+      cy.get('[class*="42-node-label"]').should('exist').should('contain', '1');
+      cy.get('[class*="41-node-label"]').should('exist').should('contain', 'Demo Project');
+      cy.deleteAwxWorkflowJobTemplate(workflowJobTemplate);
+    });
+
+    it('Should create a workflow job template and then navigate to the visualizer, and then navigate to the details view after clicking cancel', () => {
+      const jtName = 'E2E ' + randomString(4);
+      // Create workflow job template
+      cy.navigateTo('awx', 'templates');
+      cy.clickButton(/^Create template$/);
+      cy.clickLink(/^Create workflow job template$/);
+      cy.get('[data-cy="name"]').type(jtName);
+      cy.get('[data-cy="description"]').type('this is a description');
+      cy.get('[data-cy="Submit"]').click();
+      cy.get('[data-cy="workflow-visualizer"]').should('be.visible');
+      cy.get('h4.pf-v5-c-empty-state__title-text').should(
+        'have.text',
+        'There are currently no nodes in this workflow'
+      );
+      cy.get('div.pf-v5-c-empty-state__actions').within(() => {
+        cy.get('[data-cy="add-node-button"]').should('be.visible');
+      });
+      cy.get('button[data-cy="workflow-visualizer-toolbar-close"]').click();
+      cy.verifyPageTitle(`${jtName}`);
```

Comment:
Should we assert that certain details are shown as part of the details page? At least the ones added to create the workflow job template?